custom_completions_test.go

```diff
@@ -720,17 +720,17 @@ func TestRequiredFlagNameCompletionInGo(t *testing.T) {
 	rootCmd.AddCommand(childCmd)
 
 	rootCmd.Flags().IntP("requiredFlag", "r", -1, "required flag")
-	rootCmd.MarkFlagRequired("requiredFlag")
+	er(rootCmd.MarkFlagRequired("requiredFlag"))
```

Comment:
Do we need these `er` wrappers here? Since it's a test, if we are not expecting an error we should specifically assert on them.